signals/sender/sender_legitimate_domain_negation.yml

```diff
@@ -0,0 +1,27 @@
+name: "Sender: Legitimate message senders"
+type: "query"
+source: |  
+  (
+    sender.email.domain.root_domain in ("protectedtrust.com", "medallia.com", "icims.com", "workday.com")
+    or any(body.links,
+            .href_url.domain.root_domain == sender.email.domain.root_domain
+    )
```

Comment:
```suggestion
```

I'm not sure these senders _always_ send secure messages... thoughts?
as for L6-8, it's probably a remnant from pulling these signals out